dgf/templatetags/dgf.py

```diff
@@ -286,6 +286,11 @@ def days_since(date):
     return (timezone.now().date() - date.date()).days
 
 
+@register.filter
+def exceeds_bag_tag_limit(days_since):
+    return days_since > 365
```

Comment:
Könnte man die 365 über die Admin-Seite konfigurierbar machen?

Reply:
Will be done: https://github.com/thunder-guanaco/dgf/issues/7239